website/docs/r/linux_function_app.html.markdown

```diff
@@ -67,6 +67,8 @@ The following arguments are supported:
 
 * `site_config` - (Required) A `site_config` block as defined below.
 
+* `flex_function_app_deployment` - (Optional) A `flex_consumption_app_deployment` block as defined below.
```

Comment:
These documentation updates on the linux function app should be removed since these properties haven't been added to the resource.

Reply:
removed